README.md

```diff
@@ -2,6 +2,10 @@
 
 Demonstrates the value of Clean as You Code methodology
 
+## Installation
+* Download the plugin [JAR file](https://binaries.sonarsource.com/?prefix=Distribution/sonar-cayc-plugin/)
+* Copy the JAR file to the `lib/extensions/` directory of your SonarQube installation, then start the server.
```

Comment:
```suggestion
* Copy the JAR file to the `extensions/plugins/` directory of your SonarQube installation, then start the server.
```

The other is for _internal plugins_, aka analyzers.

Reply:
@philippe-perrin-sonarsource yeah, I understand that it's for internal plugins only, but this is in fact internal analyzer. 
Using `extensions/plugins/` will generate a big warning message on server start if user sure about safety using third-party plugin. I don't think that's what we want. cc @vivek-reghunath-sonarsource

Reply:
True. I remember it now. So many back and forth with this plugin 😫.